File: 001 - Fritz Buzz/FritzBuzz.py
############
# Reto: Fritzbuzz en Python
#########################################

for num in range(100):

    num3 = num % 3
    num5 = num % 5

    if num3 == 0 and num5 == 0:
        print("Fritz Buzz")
        continue

    elif num3 == 0 and num5 != 0:
        print("Buzz")
        continue

    elif num3 != 0 and num5 == 0:
        print("Fritz")
        continue

    else:
        print(num)
        continue

File: 001 - Fritz Buzz/FritzBuzz.php
<?php

	############
	# Reto: Fritzbuzz en PHP
	#########################################
	
	for($x = 1; $x <= 100; $x++) {

	  if($x % 5 == 0 && $x % 3 == 0) {
	    echo $x." Fritz Buzz!! x 3 y 5!\n";
	  }
	  else if($x % 5 == 0 && $x % 3 != 0) {
	    echo $x." Fritz x 5\n";
	  }
	  else if($x % 5 != 0 && $x % 3 == 0) {
	    echo $x." Buzz x 3\n";
	  } else {
	    echo $x."\n";
	  }
	  
	}

?>